Makefile

```diff
@@ -0,0 +1,5 @@
+test:
+	MLOCK_SECRETS=false cargo test
+
+bench:
+	MLOCK_SECRETS=false RUSTFLAGS="-C target_cpu=native" cargo bench
```

Comment:
I'd prefer not to add a makefile, and to stick to a single build system.
Not sure by how much `MLOCK_SECRETS` and `target_cpu` even influence the performance? I imagine that in any case just `cargo bench` yields useful results, too.

Reply:
Hmm, at the very least it would be missing `.PHONY`, but I believe this is a case where a one-line shell script might be a better fit?

As for `MLOCK_SECRETS`, I was not able to reproduce any performance gains in `hbbft` yet, but @DrPeterVanNostrand knows more about this in the case of "naked" `threshold_crypto`. Now, if we wanted to find out, we could... run the benchmarks, so I guess it's probably better not to hardcore this? =)

Reply:
On a new AWS m5.2xlarge instance: running `RUSTFLAGS="-C target_cpu=native" cargo bench` then `MLOCK_SECRETS=false RUSTFLAGS="-C target_cpu=native" cargo bench` I get the following benchmark changes:

```
Polynomial multiplication/10 +131.05%
Polynomial multiplication/20 +47.092%
Polynomial multiplication/40 +21.422%
Polynomial interpolation/5 +144.17%
Polynomial interpolation/20 +125.53%
Polynomial interpolation/40 +103.46%
```

I didn't run `bench` on this PR, but I would expect setting `MLOCK_SECRETS=false` would have little to no effect on these  benchmarks because they are adding and subtracting polynomials with the same number of coefficients (i.e. no additional memory locking/unlocking occurs during arithmetic; which is does happen during  polynomial multiplication).

Reply:
I went ahead and removed the Makefile.